x-pack/plugins/ingest_manager/public/applications/ingest_manager/components/settings_flyout.tsx

```diff
@@ -27,37 +27,71 @@ import { EuiText } from '@elastic/eui';
 import { useInput, useComboInput, useCore, useGetSettings, sendPutSettings } from '../hooks';
 import { useGetOutputs, sendPutOutput } from '../hooks/use_request/outputs';
 
+const URL_REGEX = /^(https?):\/\/[^\s$.?#].[^\s]*$/gm;
```

Comment:
Just curious what source of this regexp?

Reply:
From my personal gists, not sure what the original source is, it's not perfect but the url is also validated server side, with kbn/schema so it should do the work.